python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -55,52 +51,14 @@ class ApPipeConfig(pexConfig.Config):
     )
 
     def setDefaults(self):
-        """Settings assumed in baseline ap_pipe runs.
+        """Settings appropriate for most or all ap_pipe runs.
         """
-        # TODO: remove explicit DECam reference in DM-12315
-        obsDecamDir = getPackageDir("obs_decam")
-        self.ccdProcessor.load(os.path.join(obsDecamDir, "config/processCcd.py"))
-        self.ccdProcessor.load(os.path.join(obsDecamDir, "config/processCcdCpIsr.py"))
-
-        self.ccdProcessor.calibrate.doAstrometry = True
-        self.ccdProcessor.calibrate.doPhotoCal = True
-
-        # Use gaia for astrometry (phot_g_mean_mag is only available DR1 filter)
-        # Use pan-starrs for photometry (grizy filters)
-        for refObjLoader in (self.ccdProcessor.calibrate.astromRefObjLoader,
-                             self.ccdProcessor.calibrate.photoRefObjLoader,
-                             self.ccdProcessor.charImage.refObjLoader,):
-            refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
-        self.ccdProcessor.calibrate.astromRefObjLoader.ref_dataset_name = "gaia"
-        self.ccdProcessor.calibrate.astromRefObjLoader.filterMap = {
-            "u": "phot_g_mean_mag",
-            "g": "phot_g_mean_mag",
-            "r": "phot_g_mean_mag",
-            "i": "phot_g_mean_mag",
-            "z": "phot_g_mean_mag",
-            "y": "phot_g_mean_mag",
-            "VR": "phot_g_mean_mag"}
-        self.ccdProcessor.calibrate.photoRefObjLoader.ref_dataset_name = "pan-starrs"
-        self.ccdProcessor.calibrate.photoRefObjLoader.filterMap = {
-            "u": "g",
-            "g": "g",
-            "r": "r",
-            "i": "i",
-            "z": "z",
-            "y": "y",
-            "VR": "g"}
-
```

Comment:
It looks like you've moved all this refcat filterMap business into `config/apPipe.py` in `ap_verify_hits2015`. Do all our "datasets" now need to include a config like this in order to run ap_pipe on them? Could this information live in the obs_package's `config/apPipe.py` instead?